spyder/plugins/editor/widgets/base.py

```diff
@@ -454,82 +472,22 @@ def get_cell_as_executable_code(self, cursor=None):
             text = ls * line_from + text
         return text, block
 
-    def is_cell_separator(self, cursor=None, block=None):
-        """Return True if cursor (or text block) is on a block separator"""

```

Comment:
Why was all this removed? 

select_current_cell is now doing what select_current_cell_in_visible_portion was or ?

Reply:
Yes `select_current_cell_in_visible_portion` was not actually doing something different from `select_current_cell` and `is_cell_separator` was duplicated code as well.

Reply:
Got it!